python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py

```diff
@@ -231,33 +231,42 @@ def search_summary(self):
         return self.hposearcher.search_summary()
 
     def quantize(self, pl_model,  # remove the type requirement for type checking
+                 precision='int8',
+                 accelerator=None,
+                 method='fx',
                  calib_dataloader: DataLoader = None,
                  val_dataloader: DataLoader = None,
                  metric: Optional[Metric] = None,
                  backend='inc',
                  conf: Optional[str] = None,
-                 framework='pytorch_fx',
                  approach='static',
                  tuning_strategy='bayesian',
                  accuracy_criterion: dict = {'relative': 0.99, 'higher_is_better': True},
                  timeout=0,
                  max_trials=1,
-                 return_pl=True
+                 input_sample=None,
+                 return_pl=False
                  ):
         """
         Calibrate a Pytorch-Lightning model for post-training quantization.
 
-        :param pl_model:       A Pytorch-Lightning model to be quantized.
+        :param pl_model:        A Pytorch-Lightning model to be quantized.
+        :param precision:       Global precision of quantized model,
+                                supported type: 'int8', 'bf16', 'fp16', defaults to 'int8'.
+        :param accelerator:     Use accelerator 'None', 'onnxruntime', 'openvino', defaults to None.
+                                None means staying in pytorch.
+        :param method:          Method to do quantization.
```

Comment:
Could you provide a brief description of all these methods or some links for users to find more information? Such as what is "fx" or "eager".

Reply:
Added